### ISSUE TITLE
New release with new summary() etc

### ISSUE THREAD

**Issue by hlapp** [ropensci/RNeXML#241]
@cboettig can a new release be cut to CRAN? We will be running a teaching workshop on RPhenoscape and friends in conjunction with Evolution, and are planning to send installation instructions on Monday. It would be nice if participants won't have to run through dependency upgrades again once they are at Evolution (or at the start of the workshop).

**Comment by cboettig** [comment]
Hey @hlapp , apologies, I haven't quite finished #240 (takes me a while to figure out what some of the old `lazyeval` syntax was actually doing -- I see why they deprecated that!)

We may also be held up the `taxadb` Suggested dependency, which depends on MonetDBLite which was archived by CRAN and hasn't yet been replaced.  But I think we can work around that (as it's only a Suggested dependency, we can probably drop it from the DESCRIPTION; but may need to add some conditional skips to the testthat tests.

Even without these issues it's not clear we'd clear the CRAN process before Monday though...

**Comment by hlapp** [comment]
Monday would be fine, I think. I believe I already put in the conditional skipping of tests if taxadb isn’t installed.

**Comment by hlapp** [comment]
See #238.

**Comment by hlapp** [comment]
> takes me a while to figure out what some of the old `lazyeval` syntax was actually doing

BTW that's some consolation as I remember staring at that for more than an hour and just not understanding what it was going to do and why.

**Comment by hlapp** [comment]
@cboettig any hope that this can be released soon?

**Comment by cboettig** [comment]
yeah I'm a bit worried they won't like the dependency on `taxadb` which replaced the much heavier dependency on `taxize`, since it's still not on CRAN (though hopefully will get it into rOpenSci peer review soon).  It's just a soft dependency though (Suggests), so maybe they will let it slide.  I'll give it a go and see what happens.

**Comment by hlapp** [comment]
@cboettig any update here? I'm concerned we've maneuvered this into a position where new releases of the package cannot be made anymore.

FYI, we already have the next training workshop scheduled in conjunction with the SSB stand-alone meeting in Gainesville, FL, at the beginning of Jan 2020. I argue that if we cannot re-release RNeXML well in time before that, we have an untenable situation on our hands.

**Comment by cboettig** [comment]
apologies,  I think we are clear to release, I just lost track of this (CRAN tests weren't available when I tried  in August and I forgot to come back 😊 ).  lemme try again !

**Comment by hlapp** [comment]
@cboettig any update as to where we're at with this?

**Comment by cboettig** [comment]
yeah, came back to us, CRAN really would prefer the suggested dependency of `taxadb` be on CRAN first. The good news is that we're probably ready for a first release of `taxadb` anyway, it's just been in rOpenSci review; but we can probably submit it to CRAN simultaneously.

**Comment by hlapp** [comment]
@cboettig any updates? Sorry to have to keep pestering, but this issue is getting more important by the day.

**Comment by cboettig** [comment]
Thanks for bugging and sorry for letting this drag out so long.  I've just submitted a copy to CRAN.

It will get tagged for manual review since the check throws a NOTE due to the `taxadb` availability, so it will probably take a little longer to go through and no guarantees what will happen during the manual review, but I figured it is worth a shot.  (There's no reason the soft / suggested dependency should be a problem, it is after all a suggested dependency of the current CRAN version too, but manual review historically can result in complaints about other entirely unrelated things; whereas a package that goes through check with no NOTEs is nowadays usually automatically updated without further human review and objections about things outside of the automated check system... I've probably been too paranoid about that anyway so hopefully this still goes through just fine.)

meanwhile `taxadb` is just about all cleared on rOpenSci review anyway, so it will also be CRAN bound soon.
